addons/sys_core/XEH_postInit.sqf

```diff
@@ -112,42 +94,22 @@ if (GVAR(revealToAI) && hasInterface) then {
 
 [QGVAR(onRevealUnit), { _this call FUNC(onRevealUnit) }] call CALLSTACK(CBA_fnc_addEventHandler);
 
-ACRE_PLAYER_VEHICLE_CREW = [];
 //Store objects occupying crew seats, note this is empty if the player is not a crew member.
-
-private _vehicleCrewPFH = {
-    private _vehicle = vehicle acre_player;
-
-    if (_vehicle != acre_player) then {
-        private _crew = [driver _vehicle, gunner _vehicle, commander _vehicle];
-        {
-            _crew pushBackUnique (_vehicle turretUnit _x);
-        } forEach (allTurrets [_vehicle, false]);
-        _crew = _crew - [objNull];
-        if (acre_player in _crew) then {
-            ACRE_PLAYER_VEHICLE_CREW = _crew;
-        } else {
-            ACRE_PLAYER_VEHICLE_CREW = [];
-        };
-    } else {
-        ACRE_PLAYER_VEHICLE_CREW = [];
-    };
-};
-ADDPFH(_vehicleCrewPFH, 1.1, []);
+ACRE_PLAYER_VEHICLE_CREW = [];
+ADDPFH(DFUNC(vehicleCrewPFH), 1.1, []);
```

Comment:
I have seen this in several other files, specially those within the ADDPFH macro, like in fnc_enableRevealAI.sqf, fnc_handleMultiPttKeyPressUp.sqf, fnc_utilityFunction.sqf in sys_core alone. I wasn't sure about them so I kept it as they were originally written. Should all of these be then changed?